doc/charm++/manual.rst

```diff
@@ -3152,13 +3141,12 @@ Below are the descriptions about the compiler and runtime options:
    path to the library and link the load balancer into an application
    using *-module FooLB*.
 
-   You can create a library by modifying the Makefile in the following
-   way. This will create *libmoduleFooLB.a*.
+   You can create a library in the following way. This will create
+   *libmoduleFooLB.a*.
 
-   .. code-block:: makefile
+   .. code-block::
```

Comment:
Should this be `:: bash` or `:: none`?

Reply:
Is there a good way of specifying this when building with CMake (as in specifying an specific LB to be built as part of the Charm++ build command)?

Also, perhaps this should still be included under the "Building individual load balancers" heading since this also applies to non-user written LBs.

Reply:
I'm on board with building all LBs by default, I agree that the cost is low and it makes things simpler for users. The only issue is for strategies like ZoltanLB or ScotchLB that have external dependencies, which users should still have some way of easily building.

I think the code sample under "Write and use your own load balancer" was intended to be put into application Makefiles, but I'm not sure.